server/devices_test.go

```diff
@@ -1,47 +1,235 @@
 package server_test
 
 import (
+	"bytes"
+	"database/sql"
 	"net/http"
 	"net/http/httptest"
 	"strings"
 	"testing"
 
 	main "github.com/rgdaddio/gobotics/server"
+	cd "github.com/rgdaddio/gobotics/utils/clientdevices"
 	mock "github.com/rgdaddio/gobotics/utils/clientdevices/mock"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestDevicesHandler(t *testing.T) {
+	testCases := []struct {
+		name                 string
+		requestMethod        string
+		devicesToAdd         cd.Devices
```

Comment:
this typing is off. Should be []cd.Device

Reply:
I had made a alias in `clientdevices.go` 
```
// Devices - a list of multiple devices
type Devices []Device
```
should I remove it?

Reply:
oh nevermind, I misread and thought you had just cd.Device
The alias is fine.